assets/model_monitoring/components/src/shared_utilities/histogram_utils.py

```diff
@@ -43,6 +43,12 @@ def get_dual_histogram_bin_edges(
 
         bin_width = (max_value - min_value) / num_bins
 
+        # If histogram has only one value then we only need a single bucket and
+        # should skip the for-loop below.
+        if min_value == max_value:
+            all_bin_edges[col] = [min_value, min_value * 1.01]
```

Comment:
Not a big deal, but it might be slightly more intuitive for the user if the bin is centered around the single value. The easiest way to achieve this could be to compute a `delta = min_value*0.005` and then define `all_bin_edges[col] = [min_value-delta, min_value+delta]`